pallets/identity/src/types.rs

```diff
@@ -9,6 +9,7 @@ use sp_std::{prelude::*, vec::Vec};
 pub type Error = Vec<u8>;
 pub type CddStatus = Result<IdentityId, Error>;
 pub type AssetDidResult = Result<IdentityId, Error>;
+type Permissions = Vec<polymesh_primitives::Permission>;
```

Comment:
@vkomenda You should only need to switch this out for `use polymesh_primitives::Permissions;` and it should start compiling again.

Reply:
(Also the schema now)